### DISCUSSION TITLE
Clio V: "Eco" mode defaults to 'off' every time, can this be changed?

### DISCUSSION THREAD

**Discussion by x5nder** [cedricp/ddt4all#1016] [Q&A]
Renault Clio V Hybrid RS (2020);

I always use the Eco setting when driving. However, the car does not remember the active setting (on/off) but rather always defaults to 'off' when I start the car, which is quite annoying. The Renault dealership told me this is not normal behavior, but they have no idea how to change this. Does anyone have any idea if this is something that can be fixed through ddt4all?

**Comment by matzel82** [comment]
Hello,

Do you have try to configure your own profile in "my sense" ?

Is your easy link up to date ?

Have your dealer try to update with the dealers alliance update-tool ?

Possibly there is a fix.

**Comment by x5nder** [comment]
Yeah that's the strange thing, I can't find MySense anywhere... it doesn't seem to be enabled on my car. Where the Multi-Sense button is, I only have the Eco button. I wonder if maybe that can be changed with ddt4all then...

Easy Link was updated yesterday from 283C39788R (v2.9) to 283C37784R (v2.13) but no change in behavior.

**Comment by AndyIreland** [comment]
is it the MEX settings?

(sorry I have not got a Clio5 so its just guessing) 
![image](https://github.com/cedricp/ddt4all/assets/49172217/a0910b51-b9c1-4975-89ab-e15f04dd0316)

**Comment by davide-marcon** [comment]
Did you find a solution? I have the same problem and I would like to use eco mode as the default mode on my clio5

**Comment by kartoffelpflanze** [comment]
You can activate the MultiSense menu on any Easylink. Just search for "MEX" on each screen in DDT4ALL. You probably need to find an option like "MEX_HMI" that must be enabled, I don't know off the top of my head. Then somewhere like "Mode presence" you can choose which modes are available.

But be aware that you must unlock or bypass the gateway if you want to change any coding. See my post here for more details.

**Comment by x5nder** [comment]
Is there anyone here with a Clio V and MultiSense/MySense that is willing to create screenshots for me in DDT4ALL to show me the current settings of his/her car regarding all MEX parameters? You obviously can do that without unlocking the SGW, so it shouldn't be too much work 😬
